Add better documentation for MRH payments (#34)
breez/breez-sdk-liquid-docs: src/guide/send_payment.md

</section>
</custom-tabs>

<div class="warning">
<h4>Developer note</h4>

> When paying an invoice generated by another Liquid SDK instance, or any other application which uses the Boltz swapper internally, the payment will fall back to a direct onchain payment. The advantage of doing so is that the payee will spend less on fees, as they are not relying on external services to execute the payment any longer. To learn more about this process and how it works in detail, see Boltz's documentation for 

Reviewer: ```suggestion
When paying an invoice generated by another Liquid SDK instance, or any other application which uses the Boltz swapper internally, the payment will fall back to a direct onchain payment. The advantage of this is the payee will spend less on fees, as they are no longer relying on external services to execute the payment. To learn more about this process and how it works in detail, see the Boltz documentation for 
```
